lib/cartopy/tests/mpl/test_gridliner.py

```diff
@@ -510,7 +510,6 @@ def test_gridliner_remove():
     gl.remove()
 
     assert gl not in ax.artists
-    assert not ax.collections
```

Comment:
Broke this by making coastlines a collection.  However,  this line was redundant since the PR that added it also stopped `GridLiner` adding the `LineCollection`s to the axes.